retriever/lib/tools.py

```diff
@@ -10,26 +10,49 @@
 
 from builtins import str
 from builtins import input
+import csv
 import difflib
+import io
+import json
 import os
+import platform
+import shutil
 import sys
```

Comment:
Could you explain why we are removing these imports. Are they coming in from previous imports?